README.md

```diff
@@ -233,6 +233,12 @@ tls_server_name = ""
 // The method to use for pinging external nodes. Defaults to "udp" but can
 // also be set to "socket" to use ICMP (which requires root privileges).
 ping_type = "udp"
+
+// how many failed checks trigger a state update to critical
```

Comment:
Exactly. And let me give you another example stating the value of the counter:
`failures_before_critical = 2` "PASS(0), FAIL(1), FAIL(2), PASS(1), FAIL(2), FAIL(status change and counter reset)"

By decrementing instead of resetting the counter on that PASS we prevent the situation where a:

"PASS(0), FAIL(1), FAIL(2), PASS(0), FAIL(1), FAIL(2), PASS(0), FAIL(1), FAIL(2), PASS(0), FAIL(1), FAIL(2)" would never trigger the status change, which does not seem like what we would want.

Reply:
Helpful. Thank you for the clarification, particularly around the decrement behavior. That makes sense to me.